src/Component/tests/spec/State/Processor/BulkAwareProcessorSpec.php

```diff
@@ -34,7 +34,7 @@ function it_is_initializable(): void
     }
 
     function it_calls_decorated_processor_for_each_data_for_bulk_operation(
```

Comment:
I assume test case `it_calls_decorated_processor` is alright and it just about variables rename.